File: app/src/main/java/ir/touristland/Models/price/JsonMember970522.java
package ir.touristland.Models.price;

public class JsonMember970522 {
    private Date date;
    private double boardPrice;
    private double webservicePrice;
    private double extraPersonPrice;
    private double salesPrice;
    private boolean reserveStatus;

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public double getBoardPrice() {
        return boardPrice;
    }

    public void setBoardPrice(double boardPrice) {
        this.boardPrice = boardPrice;
    }

    public double getWebservicePrice() {
        return webservicePrice;
    }

    public void setWebservicePrice(double webservicePrice) {
        this.webservicePrice = webservicePrice;
    }

    public double getExtraPersonPrice() {
        return extraPersonPrice;
    }

    public void setExtraPersonPrice(double extraPersonPrice) {
        this.extraPersonPrice = extraPersonPrice;
    }

    public double getSalesPrice() {
        return salesPrice;
    }

    public void setSalesPrice(double salesPrice) {
        this.salesPrice = salesPrice;
    }

    public boolean isReserveStatus() {
        return reserveStatus;
    }

    public void setReserveStatus(boolean reserveStatus) {
        this.reserveStatus = reserveStatus;
    }

    @Override
    public String toString() {
        return
                "JsonMember970522{" +
                        "date = '" + date + '\'' +
                        ",boardPrice = '" + boardPrice + '\'' +
                        ",webservicePrice = '" + webservicePrice + '\'' +
                        ",extraPersonPrice = '" + extraPersonPrice + '\'' +
                        ",salesPrice = '" + salesPrice + '\'' +
                        ",reserveStatus = '" + reserveStatus + '\'' +
                        "}";
    }
}

File: app/src/main/java/ir/touristland/Models/price/JsonMember970522.kt
package ir.touristland.Models.price

class JsonMember970522 {
    var date: Date? = null
    var boardPrice: Double = 0.toDouble()
    var webservicePrice: Double = 0.toDouble()
    var extraPersonPrice: Double = 0.toDouble()
    var salesPrice: Double = 0.toDouble()
    var isReserveStatus: Boolean = false

    override fun toString(): String {
        return "JsonMember970522{" +
                "date = '" + date + '\''.toString() +
                ",boardPrice = '" + boardPrice + '\''.toString() +
                ",webservicePrice = '" + webservicePrice + '\''.toString() +
                ",extraPersonPrice = '" + extraPersonPrice + '\''.toString() +
                ",salesPrice = '" + salesPrice + '\''.toString() +
                ",reserveStatus = '" + isReserveStatus + '\''.toString() +
                "}"
    }
}
